test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -254,6 +254,46 @@ public class IRNode {
         vectorNode(ADD_VL, "AddVL", TYPE_LONG);
     }
 
+    public static final String SADD_VB = VECTOR_PREFIX + "SADD_VB" + POSTFIX;
```

Comment:
```suggestion
    public static final String SATURATING_ADD_VB = VECTOR_PREFIX + "SATURATING_ADD_VB" + POSTFIX;
```
I would prefer if this was written out.

Reply:
Ah. Now I see that we also named it `VectorOperators.SADD`... hmm. Would have been nice to say what the `S` means here. I would always go with the longer, more explicit name (unless it is really very very long).